scripts/rofl-puzzles.lic

```diff
@@ -0,0 +1,851 @@
+=begin
+  version: 0.1
+
+  Rings of Lumnis Puzzle Solver
+
+  Usage:
+    ;rofl-puzzles <puzzle name>
+
+  where <puzzle name> is one of the following:
+
+    auto       automatically solve puzzles when they appear (default behavior)
+
+    statue     move some some statues around on an altar
+    mosaic     find the right gem and symbol to put into the mosaic
+    crystal    choose the correct damage type to destroy monster paintings
+    stars      select the correct constellation to place in the night sky
+    ghost      use a wand to do stuff to a ghost
+    scramble   guess the word from scrambled letters
+    symbol     paint an Arkati symbol the correct colors and assemble
+    box        get some stuff from a box and do some things
+    candle     light the correct candle
+
+  contributions/thanks:
+    * Alastir for creating ;RoL to answer trivia questions and providing
+      solutions to many anagrams and the ghost, box, and stars puzzles
+    * Cigger for the gem anagram answers and testing
+    * Naamit and Maetriks for creature anagrams
+    * Claudaro for help testing and providing feedback
+    * Roblar for some hints to puzzle solutions
+    * Many other people in the HOA #puzzle_spoilers channel for answers and feedback
+
+  **Pull Requests Welcome** - you can contribute changes at:
+  https://github.com/elanthia-online/scripts/tree/master/scripts/rofl-puzzles.lic
+
+  Author: Kragdruk
+  Tags: festival, RoL, Rings
+=end
+
+module RofL
+  class Statue
```

Comment:
I would have really preferred a way to split this up into separate files for each puzzle class. For some of my personal scripts that I haven't published, I've worked around this using load `./other_file.lic` but that requires `;trust`ing all the things which is not worth it for stuff I want to share in the repo.